cogs/help.py

```diff
@@ -89,23 +92,39 @@ async def create_help_thread(self, interaction: Interaction) -> None:
         close_button_view._thread_author = interaction.user
 
         type_to_colour: Dict[str, Colour] = {
-            "Nextcord": Colour.red(),
+            "Nextcord": Colour.blurple(),
             "Python": Colour.green()
         }
 
         em = Embed(
             title = f"{self._help_type} Help needed!",
-            description = f"Alright now that we are all here to help, what do you need help with?",
+            description = f"What do you need help with?",
             colour = type_to_colour.get(self._help_type, Colour.blurple())
         )
-        em.set_footer(text = "You and the helpers can close this thread with the button")
+        em.set_footer(text = "You and the help moderators can close this thread with the button")
 
         msg = await thread.send(
-            content = f"<@&{HELPER_ROLE_ID}> | {interaction.user.mention}",
+            content = interaction.user.mention,
             embed = em,
             view = ThreadCloseView()
         )
         await msg.pin(reason = "First message in help thread with the close button.")
+        return thread
+
+    async def __launch_wait_for_message(self, thread: Thread, interaction: Interaction) -> None:
+        def yeet(message: Message) -> bool:
```

Comment:
This should be named more professionally

Maybe `is_allowed`

Reply:
ty, i couldn't think of anything other than `check`.